packages/react-examples/src/charting/VerticalStackedBarChart/VerticalStackedBarChart.Styled.Example.tsx

```diff
@@ -118,8 +128,8 @@ export class VerticalStackedBarChartStyledExample extends React.Component<{}, IV
                 },
               },
             }}
-            // eslint-disable-next-line react/jsx-no-bind
-            onRenderCalloutPerDataPoint={props =>
+            // eslint-disable-next-line react/jsx-no-bind @typescript-eslint/no-explicit-any
```

Comment:
I think for disabling multiple rules, eslint expects you to use a comma between them.
```suggestion
            // eslint-disable-next-line react/jsx-no-bind, @typescript-eslint/no-explicit-any
```

Reply:
Done,
Thank you!!!